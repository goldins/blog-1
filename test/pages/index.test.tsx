import { render } from '../testUtils';
import Home from '../../pages';

describe('Home page', () => {
  it('matches snapshot', () => {
    // TODO: mock Date :)
    const { asFragment } = render(<Home />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
