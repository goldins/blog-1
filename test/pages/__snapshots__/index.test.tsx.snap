// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-12mft5n"
  >
    <main>
      <h1
        class="css-pavrw7"
      >
        Simon.Goldin
      </h1>
    </main>
    <a
      class="css-1wmzfz"
    >
      Blog
    </a>
    <a
      class="css-1wmzfz"
    >
      Projects
    </a>
    <footer
      class="css-8u3q9h"
    >
      <p
        class="css-0"
      >
        Footer
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
